CRM/Contact/BAO/GroupContactCache.php

```diff
@@ -173,8 +173,12 @@ protected static function updateCacheTime($groupID, $processed, ?float $startTim
       $now = date('YmdHis');
       if ($startTime) {
         $took = microtime(TRUE) - $startTime;
-        if ($took > 5) {
-          Civi::log()->warning("Updating smart group $groupID took over 5s (" . number_format($took, 3) . "s)");
+        if (defined('CIVICRM_SLOW_SMART_GROUP_SECONDS') && $took > CIVICRM_SLOW_SMART_GROUP_SECONDS) {
```

Comment:
if you use CRM_Utils_Constant to access the define then it can be set with `env` - eg https://github.com/eileenmcnaughton/org.wikimedia.systemtools/blob/master/README.md?plain=1#L44